src/components/Description.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Switch from 'react-switch';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Col,
+  Row,
+} from 'reactstrap';
+
+const Description = ({
+  t,
+  themeColor,
+  handleTitle,
```

Comment:
`handleTitle` should be `toggleTitle`, right? Because it is bound to an `onChange` handler.

Reply:
I just saw below that it indeed just toggles (show/hide) the title, so it should probably be called that.